assets/l10n/intl_en.arb

```diff
@@ -2676,5 +2676,8 @@
     },
     "today": "Today",
     "yesterday": "Yesterday",
-    "adminPanel": "Admin Panel"
+    "adminPanel": "Admin Panel",
+    "rejectInvite" : "No thanks, delete",
+    "acceptInvite" : "Yes please, join",
+    "askToInvite": " wants you to join this chat. What do you say?"
```

Comment:
```suggestion
    "askToInvite": "{displayName} wants you to join this chat. What do you say?"
```

Reply:
In this case `displayName` need highlights

Reply:
I think we should support https://pub.dev/packages/flutter_markdown for easy define l10n
in some language, displayName is not from start, it can be in middle of sentence
so when we support markdown, it should be
```
*{displayName}* wants you to join this chat. What do you say?
```